tests/unit/CommentModelTest.php

```
@@ -0,0 +1,107 @@
+    protected $model;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->model = new CommentModel($this->db);
+        $this->tenant = fake(TenantModel::class, ['subdomain' => 'acme', 'name' => 'Acme']);
+        $this->category = fake(CategoryModel::class, ['tenant_id' => $this->tenant->id]);
+        $this->user = fake(UserModel::class, ['tenant_id' => $this->tenant->id]);
+        $this->post = fake(PostModel::class, [
+            'tenant_id'   => $this->tenant->id,
```

PHPStan (2.2.1)

[error] Cannot access property $id on array|object.


(property.nonObject)